client/app/bundles/comments/components/CommentBox/CommentBox.scss

```diff
@@ -0,0 +1,21 @@
+:global {
```

Comment:
Actually no, it's a quick workaround to make `ReactCSSTransitionGroup` work. I need to change it to local classes and define them in `transitionName` prop as an object.